src/deploy/osp_deployer/settings_sanity.py

```diff
@@ -128,6 +135,23 @@ def check_ipmi_to_nodes(self):
                                         self.settings.ipmi_user,
                                         self.settings.ipmi_password)
 
+    def check_mtu_of_networks(self):
+        shouldbbevalidmtu = [
+            'mtu_size_global_default',
+            'public_api_network_mtu',
+            'private_api_network_mtu',
+            'storage_network_mtu',
+            'provisioning_network_mtu',
+            'storage_cluster_network_mtu',
+            'tenant_tunnel_network_mtu',
+            'tenant_network_mtu',
```

Comment:
Shouldn't we have checks here for floating_ip_network_mtu and management_network_mtu?

Reply:
sure will add the checks